roles/avans/templates/student_project_backup.sh.j2

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+exit_needed=$(False)
+{% for project in atgm_projects %}
+    {% if project.host == ansible_nodename %}

```

Comment:
are you sure?

I thought that this script was running on every machine.
So if the machine does not hold any student projects it will do nothing, or only does the correct ones